scarb/tests/package.rs

```diff
@@ -1190,3 +1205,55 @@ fn no_lib_target() {
         +   [lib]
         "#});
 }
+
+#[test]
+fn error_on_verification() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("foo")
+        .version("1.0.0")
+        .src("src/lib.cairo", ".")
+        .build(&t);
+
+    Scarb::quick_snapbox()
+        .arg("package")
+        .current_dir(&t)
+        .assert()
+        .failure()
+        .stdout_matches(indoc! {r#"
+        [..] Packaging foo v1.0.0 [..]
+        [..] Verifying foo-1.0.0.tar.zst
+        [..] Compiling foo v1.0.0 ([..])
+        error: Skipped tokens. Expected: [..]
+         --> [..]
+        .
+        ^
+
+        
+        error: failed to verify package tarball
```

Comment:
hmm, that's a lot of `\n`s. don't we happen to by mistake add more newlines than needed?

Reply:
Seems like that's how Cairo compiler reports it.